kpi/utils/spectacular_processing.py

```diff
@@ -0,0 +1,9 @@
+def pre_processing_filtering(endpoints):
+
+    filtered = []
+    for (path, path_regex, method, callback) in endpoints:
+        # Remove all endpoints not in 'api/v2/'
+        if not path.startswith("/api/v1/"):
```

Comment:
Use single quotes for string. You can use `./format-python.sh <base branch>` to fix all linter errors.